robot/impl/resource_manager.go

```diff
@@ -182,9 +182,14 @@ func (manager *resourceManager) updateRemoteResourceNames(
 	rr internalRemoteRobot,
 	recreateAllClients bool,
 ) bool {
-	manager.logger.CDebugw(ctx, "updating remote resource names", "remote", remoteName)
+	manager.logger.CDebugw(ctx, "updating remote resource names", "remote", remoteName, "recreateAllClients", recreateAllClients)
 	activeResourceNames := map[resource.Name]bool{}
 	newResources := rr.ResourceNames()
+	if newResources == nil {
```

Comment:
This is also part of the "don't close video streams because of a chained remote network blip". As per robot client changes, reverting to do it in a separate ticket.

I do want to keep the pkg/errors -> errors changes though. So I'm undoing these edits by hand, could use a double check that I didn't change logic here (I'm also keeping the "ok" variable name change to "nodeAlreadyExists" -- nodes not existing is also OK!)